apps/docs/src/app/core/component-docs/object-marker/examples/object-marker-example.component.html

```diff
@@ -1,5 +1,5 @@
 <span fd-object-marker glyph="add-favorite" title="Favorite" aria-label="Favorite Icon"></span>
-<span fd-object-marker glyph="flag" title="Flag" aria-label="Favorite Flag"></span>
-<span fd-object-marker glyph="request" title="Request" aria-label="Favorite Request"></span>
-<span fd-object-marker glyph="user-edit" title="Editable" aria-label="Favorite Editable"></span>
-<span fd-object-marker glyph="private" title="Private" aria-label="Favorite Private"></span>
+<span fd-object-marker glyph="flag" title="Flag" aria-label="Flag Icon"></span>
+<span fd-object-marker glyph="request" title="Request" aria-label="Request Icon"></span>
+<span fd-object-marker clickable="true" glyph="user-edit" title="Editable" aria-label="Editable Icon"></span>
+<span fd-object-marker clickable="true" glyph="private" title="Private" aria-label="Private Icon"></span>
```

Comment:
should not be clickable. They are not links

Reply:
Modified as link.